website/source/docs/commands/snapshot/restore.html.markdown.erb

```diff
@@ -19,7 +19,7 @@ intended to be used when recovering from a disaster, restoring into a fresh
 cluster of Consul servers.
 
 If ACLs are enabled, a management token must be supplied in order to perform
-snapshot a snapshot save.
+snapshot a snapshot restore.
```

Comment:
```suggestion
a snapshot restore.
```

Reply:
Ha I didn't even notice that. It's also wrong on the save page.